docs/examples/graph_kernels.py

```diff
@@ -134,9 +134,23 @@
 
 # %%
 likelihood = gpx.Gaussian(num_datapoints=D.n)
-prior = gpx.Prior(mean_function=gpx.Zero(), kernel=gpx.GraphKernel(laplacian=L))
+kernel = gpx.GraphKernel(laplacian=L)
+prior = gpx.Prior(mean_function=gpx.Zero(), kernel=kernel)
 posterior = prior * likelihood
 
+# %% [markdown]
+#
+# For researchers and the curious reader, GPJax provides the ability to print the
+# bibtex citation for objects such as the graph kernel through the `cite()` function.
+
+# %%
+print(gpx.cite(kernel))
+
```

Comment:
I have a couple of questions on citations, and am using this one as an example.

calling the cite function on the graph kernel gives:
```
 @inproceedings{borovitskiy2021matern,
authors = {Borovitskiy, Viacheslav and Azangulov, Iskander and Terenin, Alexander and Mostowsky, Peter and Deisenroth, Marc and Durrande, Nicolas},
title = {Matérn Gaussian Processes on Graphs},
year = {2021},
booktitle = {International Conference on Artificial Intelligence and Statistics},
}
```

My questions are:

- (1) `Gaussian` in `title` should be capitalised, but this won't necessarily be capitalised in Latex by default, is capitalisation something we want to enforce for all the citations?
- (2) are things like the `é`  in `Matérn` in `title` latex friendly?

Reply:
1) I would always recommend people to have `\DeclareFieldFormat{titlecase}{#1}` in their preamble to spare having to have lots of {} pairs in your bib file as they make the reading significantly messier. I can add {} if you feel strongly that we need them here.
2) This is fine in my experience.